File: backend/src/main/java/top/vuhe/entity/equipment/dao/ThresholdDao.java
package top.vuhe.entity.equipment.dao;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.vuhe.entity.BaseEntity;

/**
 * @author zhuhe
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class ThresholdDao extends BaseEntity {
    private Integer channelId;
    private Integer statusId;
    private Integer thresholdValue;
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/entity/equipment/dao/ThresholdDAO.kt
package top.vuhe.sw.entity.equipment.dao

/**
 * 阈值信息
 * 对应数据库中的表
 *
 * @property id             id
 * @property channelId      通道id
 * @property statusId       状态id
 * @property thresholdValue 阈值
 */
data class ThresholdDAO(
    val id: Int?,
    val channelId: Int?,
    val statusId: Int?,
    val thresholdValue: Int?,
)
